docs/agents/index.md

```diff
@@ -124,3 +124,91 @@ async def main():
 if __name__ == "__main__":
     asyncio.run(main())
 ```
+
+### Multi-Turn Conversations
```

Comment:
💅 can be in a follow-up: This feels worth a separate page. Not sure where. Maybe we should add a new "How-to" section where we cover specific scenarios